LICENSE

```diff
@@ -1,207 +1,7 @@
+Copyright 2020 Jens Axboe
```

Comment:
You shouldn't change the FDB license here ;)

Reply:
Hi Markus, we are going to add liburing as an external project and have it downloaded, compiled and linked at make time. I think this should address the issue. If not, please let me know. Thank you.

Reply:
sorry for my high latency.

Adding a license file for a new library is fine. But this replaces the FDB license file. Instead this license should probably go to the `contrib` directory

Reply:
I have added liburing's license to the ACKNOWLEDGEMENTS file (same thing I did for another PR).
Note that liburing is not part of the codebase: it is downloaded on the fly (fdbrpc/CMakelists).

Reply:
@sfc-gh-mpilman Hi Markus, what's the next step here? Are you waiting for something specific on our side? Thx